mlflow/models/evaluation/base.py

```diff
@@ -1061,11 +1061,12 @@ def predict(self, context, model_input: pd.DataFrame):
 
 
 def evaluate(
-    model: str,
-    data,
+    model: str = None,
+    data=None,
```

Comment:
By doing this:

1. Existing calls to evaluate where model and data are passed positionally will continue to work.
2. Users can omit passing model and data, in which case they'll be set to None.
3. The model_type remains a mandatory keyword-only argument.
4. env_manager retains its default value of "local" if not provided by the caller.

Reply:
these make sense to me. btw, we are planning to make `model_type` optional as well in https://databricks.atlassian.net/browse/ML-35422